doc/HOWTO_Import_Geodata.md

```diff
@@ -0,0 +1,12 @@
+# How to Import the Third Party Geodata
+
+The Geodata zip contains 2 sql files, a small one with community names, and a large one which includes streets.
+
+1. Copy the smaller sql file into /build/database.
+2. Rename it to `00.GEodata.sql`. This is needed because Docker runs the files in order, and the Geodata import needs to run before the Schema.
```

Comment:
```suggestion
2. Rename it to `00.Geodata.sql`. This is needed because Docker runs the files in order, and the Geodata import needs to run before the Schema.
```